jobserver/models.py

```diff
@@ -487,3 +488,37 @@ def repo_name(self):
         if not f.path:
             raise Exception("Repo URL not in expected format, appears to have no path")
         return f.path.segments[-1]
+
+
+class Release(models.Model):
```

Comment:
Super personal nit (sorry)… can the alphabetical ordering of models be kept please?  (it really helps me scan down the list!)